code/upaya/config/initializers/omniauth.rb

```diff
@@ -10,7 +11,9 @@
   certificate: Rails.application.secrets.saml_cert,
   private_key: OpenSSL::PKey::RSA.new(
     File.read(Rails.root + 'config/saml.key.enc'),
-    Figaro.env.saml_passphrase).to_pem,
+    Figaro.env.saml_passphrase || '').to_pem,
+  private_key: OpenSSL::PKey::RSA.new(
+    File.read(Rails.root + 'config/saml.key.enc')).to_pem,
```

Comment:
`private_key` is present twice. Which is the correct one? The one with the `saml_passphrase` or the one without?